program-data-separation/cpp/main.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/extension/module/module.h>
```

Comment:
Do we need to update this doc? 

https://docs.pytorch.org/executorch/main/extension-module.html